### PR TITLE
Create package

### PR DESCRIPTION
**Pull request by eslavich**
Initial batch of migrated schemas, moved to asdf-format.org with version 2.0.0.  Includes schema tests with the existing plugin but no docs yet.

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+license = BSD 3-Clause
+license_file = LICENSE
+description = ASDF schemas for transforms
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/spacetelescope/asdf-transform-schemas
+
+[options]
+python_requires = >=3.6
+zip_safe = false
```
Comment (jdavies-st) [inline setup.cfg:14]: Is there a reason the package is not zip-safe?

Comment (eslavich) [reply setup.cfg:14]: I implemented the extension in "asdf classic" style, which means it returns a filesystem path to the location of the schema file in the installed package.  If the package were installed and loaded as a single zipped file, then the path wouldn't exist.

**Comment by eslavich** [comment]
> Do we need to add a `pyproject.toml` indicating the need for `setuptools` and `setuptools_scm`?

Hmm!  I didn't know about that.  After reading about PEP 518 it seems like a good idea.

**Comment by eslavich** [comment]
That last commit removed the three deprecated transform schemas (domain, label_mapper, regions_selector)

**Pull request merged**